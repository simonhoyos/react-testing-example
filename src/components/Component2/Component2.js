import React from 'react';

export function Component2() {
  return <h2>Componente 2</h2>;
}
