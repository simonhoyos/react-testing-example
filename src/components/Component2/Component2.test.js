import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Component2 } from './Component2';

describe('<Component2 />', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Component2 />);

    expect(toJson(wrapper)).toMatchSnapshot();
  })
})
