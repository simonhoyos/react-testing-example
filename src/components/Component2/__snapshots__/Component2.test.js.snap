// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Component2 /> should match snapshot 1`] = `
<h2>
  Componente 2
</h2>
`;
